media/css/firefox/whatsnew/whatsnew-117-relay.scss

```diff
@@ -0,0 +1,228 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import 'includes/base';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-relay';
+
+
+.wnp-content-main {
+    background-color: $color-pink-05;
```

Comment:
With the colored background I think it could also use a small top margin to add a little more whitespace under the green bar.

![image](https://github.com/mozilla/bedrock/assets/205591/8d62acf4-7a41-462f-b899-7697c05fe481)